pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -46,8 +46,8 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/framework"
 	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/feature"
 	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/names"
-	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/volumebinding"
 	schedutil "k8s.io/kubernetes/pkg/scheduler/util"
+	"k8s.io/kubernetes/pkg/scheduler/util/assumecache"
```

Comment:
Let's move all the `k8s.io/kubernetes/*` to this block since we knifed.

Reply:
Can your rephrase? I'm not sure I follow.

All `k8s.io/*` imports are currently one block.

Reply:
Sorry, I just messed up with the github diff. I thought you want to split `k8s.io/kubernetes` and `k8s.io/<pkg>` apart, forget it.